docs/source/nodes_and_pipelines/pipeline_introduction.md

```diff
@@ -236,3 +236,19 @@ The output is as follows:
 Circular dependencies exist among these items: ['first node: <lambda>([x]) -> [y]', 'second node: <lambda>([y]) -> [x]']
 ```
 </details>
+
+### Pipeline with improperly named nodes
+Nodes named with improper characters, such as the dot notation, may exhibit strange behaviour than intended.
+
+<details>
+<summary><b>Click to expand</b></summary>
+
+```python
+pipeline([node(lambda x: x, inputs: "input1", outputs: "outputs1.kedro")])
+```
+
+The output tends to be a disconnected pipeline and / or improperly formatted Kedro structure. 
```

Comment:
```suggestion
The output tends to be a disconnected pipeline and/or improperly formatted Kedro structure. 
```

Reply:
Is the actual node name "outputs1.kedro" or is that the name of the output of a node?

Reply:
Cos it would be better to be precise if it's the latter and say something like "Nodes that are created with input or output names that contain improper characters..." above if the latter.